crawler_copa_brasil.py

```diff
@@ -0,0 +1,46 @@
+from requests import get
+from bs4 import BeautifulSoup as bs
+from selenium import webdriver
+from time import sleep
+
+def print_matches(boxes):
+    for box in boxes:
+        info = box.find('div', {'class':'placar-jogo-informacoes'}).text
+
+        mandante = box.find('span', {'class':'placar-jogo-equipes-item placar-jogo-equipes-mandante'})
+        mandante_sigla = mandante.find('span', {'class':'placar-jogo-equipes-sigla'}).text
+        mandante = mandante.find('span', {'class':'placar-jogo-equipes-nome'}).text
+
+        visitante = box.find('span', {'class':'placar-jogo-equipes-item placar-jogo-equipes-visitante'})
+        visitante_sigla = visitante.find('span', {'class':'placar-jogo-equipes-sigla'}).text
+        visitante = visitante.find('span', {'class':'placar-jogo-equipes-nome'}).text
```

Comment:
Could you please use English variables here?

Reply:
Of course!